src/renderer/views/Playlist/Playlist.js

```diff
@@ -241,6 +241,20 @@ export default defineComponent({
     sortBySelectValues() {
       return this.sortByValues
     },
+    sortIcon: function () {
+      switch (this.sortOrder) {
+        case SORT_BY_VALUES.DateAddedNewest:
+        case SORT_BY_VALUES.VideoTitleDescending:
+        case SORT_BY_VALUES.AuthorDescending:
```

Comment:
I think anything with text sorting should use another icon
https://fontawesome.com/v5/search?q=sort&o=r&m=free
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/f2ad8410-1dcd-4ee8-98d4-5d1a48c3091f)

Reply:
I considered this possibility. I'll do it if we think the transformation is not too jarring from one value to another

Reply:
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/5ea3c0c8-d4bc-45c8-9856-bf5f7296cd46)
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/7aefa0a5-756f-4bfd-bd78-f768a942951d)